### PR TITLE
Ahmet week2

### PR DESCRIPTION
**Pull request by ahmthsrvsmz**

### CODE REVIEW

File: Week2/homework/index.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        console.log(error);
+        res.send(error);
+      } else {
+        res.send(results);
+      }
+    },
+  );
+});
+
+app.get(`/countries/:languages`, (req, res) => {
```
Comment (mogaleaf) [inline Week2/homework/index.js]: This way work but only if the user is adding data from your program. The user would still be able to add a new country using the sql command line for instance. That is why mysql provides a trigger. Can you try to write one instead?

File: Week2/homework/index.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+});
+
+connection.connect();
+
+const data = require('./cities.json');
+const { countries } = data;
+const { cities } = data;
+
+function getCapitalCity(id, req, res) {
+  connection.query(`select * from cities where id = "${id}"`, (error, results, fields) => {
```
Comment (mogaleaf) [inline Week2/homework/index.js]: Avoid injecting user data like that :"${id}". Use prepared statement instead ( where id = ? ).

Comment (ahmthsrvsmz) [reply Week2/homework/index.js]: 👍

File: Week2/homework/index.js (hunk 1 above)
Comment (ahmthsrvsmz) [reply Week2/homework/index.js]: 👍

**Comment by Tjebbee** [comment]
@mogaleaf ; is this one ready as well?

**Comment by Tjebbee** [comment]
Please @ahmthsrvsmz , take a look at this

**Pull request closed without merging**